module-4/backend/week_three.md

```diff
@@ -5,15 +5,69 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. At a high level, what is Node?
+```
+Node is a JS environment that allows for asynchronous processing of tasks.
+```
+
 2. What is Express? Why is Express similar to in the Ruby world?
+```
+Express:JavaScript::Rails:Ruby
+It is a framework to help you organize your application. Except, without all the support that Rails provides. Which means you get to make up your own rules.
```

Comment:
yup! i'd also say that Express is more similar to Sinatra than Rails - partly because it doesn't come with so much stuff.